terraform/codebuild-preview/provider.tf

```diff
@@ -0,0 +1,18 @@
+#---
+# Provider Configuration
+#---
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+terraform {
+  required_version = "~> 0.11"
```

Comment:
this conflicts with the last file in the PR which specifies >=0.12

Reply:
:+1: